plugin/output/file/file.go

```diff
@@ -254,12 +272,13 @@ func (p *Plugin) sealUp() {
 	oldFile := p.file
 	p.mu.Lock()
 	p.createNew()
-	p.nextSealUpTime = time.Now().Add(p.config.RetentionInterval_)
+	p.fileIsFully = false
```

Comment:
what do you think about move this (initialization of p.fileIsFully and p.fileSize) in createNew ?